addons/hr_expense/tests/test_expenses_mail_import.py

```diff
@@ -212,3 +213,18 @@ def assertParsedValues(subject, currencies, exp_description, exp_amount, exp_pro
             self.product_a,
             self.company_data['currency'],
         )
+
+
+    def test_import_expense_from_mail_get_default_expense_sheet_values_errors(self):
+        # error no product
+        message = {
+                        'message_id': "the-world-is-a-ghetto",
+                        'subject': 'no product code 800',
+                        'email_from': self.expense_user_employee.email,
+                        'to': 'catchall@yourcompany.com',
+                        'body': "Don't you know, that for me, and for you",
+                        'attachments': [],
+                    }
```

Comment:
Indent is really weird
```suggestion
        message = {
            'message_id': "the-world-is-a-ghetto",
            'subject': 'no product code 800',
            'email_from': self.expense_user_employee.email,
            'to': 'catchall@yourcompany.com',
            'body': "Don't you know, that for me, and for you",
            'attachments': [],
        }
```